css/properties/border-image-width.json

```diff
@@ -57,7 +57,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "≤72"
+                "version_added": "22"
```

Comment:
Supported with property version?

```suggestion
                "version_added": "13"
```

Reply:
Tested with [this example](http://bcd-examples.gh.caugner.de/css/border-image-width/):

- Firefox 13 did NOT support `auto` yet (note that only `-moz-border-image` was supported then):
  <img width="592" alt="image" src="https://github.com/user-attachments/assets/db7e56be-3621-40e4-b31b-f232a8a7e11c">
- Firefox 15 started supporting it:
  <img width="592" alt="image" src="https://github.com/user-attachments/assets/c105aa88-d59a-4b4d-a018-6c19f4416021">

```suggestion
                "version_added": "15"
```

PS: Just in case, I also checked that Firefox 13 didn't support `-moz-border-image-width` or `-moz-auto`.